### ISSUE TITLE
no superuser access

### ISSUE THREAD

**Issue by KocWozniakPiotr** [RandomCoderOrg/fs-manager-udroid#35]
When trying to upgrade or install any package on  fresh installation I get this error:

`dpkg: error: requested operation requires superuser privilege`
`E: Sub-process /usr/bin/dpkg returned an error code (2)`

Does anyone managed to fix it or is there a special parameter to edit when login into ubuntu?

I' m using a fresh jummy:raw version

**Comment by SaicharanKandukuri** [comment]
@KocWozniakPiotr 
considering your description either its a `sudo` or its a non-root user

> can you describe how to reproduce it

**Comment by KocWozniakPiotr** [comment]
Thanks. I used the instructions from the README so I guess nothing wrong on my side. Except maybe my phone ?
It's unrooted Xperia Z5 Compact and also I used the Termux version from f-droid.org , not the one from Playstore.

For the time being I'm just using plain Termux environment. I just needed few packages like openssh, python and apache and it works without Ubuntu anyway.

It would be cool if Ubuntu worked out of the box but it seems the image still needs a heavy overhaul to be compatible with Termux. 

The only way I see it, would be somehow simulate the whole sudo so Ubuntu thinks it's using the real thing but it doesn't look like an easy task to achieve.
